build.gradle

```diff
@@ -324,6 +367,13 @@ publishing {
 
 			// See below the publishing block for intermediary artifact publishing
 		}
+
+		// for gametest project
+		gametestable(MavenPublication) {
+			groupId 'org.quiltmc'
+			artifactId 'gametestable-quilt-mappings'
+			from components.gametestable
+		}
```

Comment:
Is there a reason the gametest project can't just consume the _normal_ publication published to that local maven? After all, it's all running in an earlier build step so you don't actually need the component/consumable variant stuff.

Reply:
Oh was the component only required for dependency substitution?
I'll try without it.

Reply:
Unless I made a mistake, it seems it needs the component publishing for gametest to resolve it
<https://github.com/supersaiyansubtlety/quilt-mappings/tree/gametest-verification-no-component-publishing>

>FAILURE: Build failed with an exception.
> 
> * What went wrong:
> A problem occurred configuring root project 'gametest'.
> > Could not resolve all dependencies for configuration ':mappings'.
   > Could not find org.quiltmc:gametestable-quilt-mappings:1.21.5+build.local.
     Searched in the following locations:
       ...
       - file:/C:/Users/super/Projects/Minecraft/Modding/Quilt/quilt-mappings/build/gametestable/org/quiltmc/gametestable-quilt-mappings/1.21.5+build.local/gametestable-quilt-mappings-1.21.5+build.local.pom
     Required by:
         root project :

Reply:
oh duh, I forgot to remove `gametestable-` from the dependency